src/pudl/transform/eia860.py

```diff
@@ -108,12 +101,21 @@ def ownership(eia860_dfs, eia860_transformed_dfs):
     own_df = pd.concat([without_known_dupes, deduped])
     # Check whether we have truly deduplicated the dataframe.
     remaining_dupes = own_df[own_df.duplicated(subset=own_pk, keep=False)]
-    if len(remaining_dupes) > 0:
+    if not remaining_dupes.empty:
         raise ValueError(
             "Duplicate ownership slices found in ownership_eia860:"
             f"{remaining_dupes}"
         )
 
+    # Remove a couple of records known to have null values in the primary key
+    mask = (
+        (own_df.report_date.isin(["2018-01-01", "2019-01-01"]))
+        & (own_df.plant_id_eia == 62844)
+        & (own_df.owner_utility_id_eia == 62745)
+        & (own_df.generator_id == "nan")
+    )
+    own_df = own_df[~mask]
+
```

Comment:
It could be nice to reference the original issue in a comment here for some justification as to why we are dropping these records. 

I think ETLs for messy data can accumulate small transformations that lack documentation. Maybe in the future, we can create notebooks tagged with data and code versions that explain why we do certain transformations.

Reply:
Yeah, these kinds of magic numbers are awful.  I'd really like to store these kinds of targeted drops/fixes/overrides in a standard format like a static dataframe that's defined in the module or maybe a data file, where we can have a comment associated with each group explaining why it's being applied. If / when we are overhauling the transform functions to make them work better with the new harvesting process it would be good try and pull out different classes of manipulations like this, and try to separate the data (which rows and/or columns they get applied to) from the operations that are being applied.

Up to this point it's been very ad-hoc, and we haven't tried to identify generalizations or patterns for reuse.

Reply:
Added a ref to the issue though for clarity / explanation.

Reply:
Sounds like a plan. Thanks for adding the issue.